packages/aila/src/features/threatDetection/detectors/lakera/LakeraThreatDetector.ts

```diff
@@ -0,0 +1,226 @@
+import { aiLogger } from "@oakai/logger";
+import { z } from "zod";
+
+import {
+  type ThreatCategory,
+  type ThreatDetectionResult,
+  type ThreatSeverity,
+  AilaThreatDetector,
+} from "../AilaThreatDetector";
+import type { Message, LakeraGuardResponse, BreakdownItem } from "./schema";
+import { lakeraGuardRequestSchema, lakeraGuardResponseSchema } from "./schema";
+
+const log = {
```

Comment:
AI Logger here with a new namespace?

Reply:
(and elsewhere we log to console)

Reply:
Ah yes. I was having trouble seeing output. Will flip this back to the ai logger